### ISSUE TITLE
Add option for navbar icons

### ISSUE THREAD

**Issue by bitboxer** [adriankarlen/textfox#78]
It would be amazing to be able to have this setting optional:

```css
#userContext-icons,
#translations-button-icon,
#tracking-protection-icon-container,
#star-button-box,
 .urlbar-page-action {
```

I prefer to see those icons, maybe with an added tint of the current theme?

**Comment by jacks-sam1010** [comment]
Hello, we tried to solve the issue.

This is what we did:

Modified urlbar.css to make icon visibility optional and add a tint of the current theme. Added a new configuration variable in config.css for icon visibility.

You can review changes in this commit: https://github.com/jacks-sam1010/adriankarlen-textfox-78/commit/f6a671eddbfdcb2d930948bdca4aba140d3f96fa.


> [!CAUTION]
> Disclaimer: The concept of solution was created by AI and you should never copy paste this code before you check the correctness of generated code. Solution might not be complete, you should use this code as an inspiration only.

---

Latta AI seeks to solve problems in open source projects as part of its mission to support developers around the world. Learn more about our mission at https://latta.ai/ourmission . If you no longer want Latta AI to attempt solving issues on your repository, you can block this account.

**Comment by bitboxer** [comment]
@jacks-sam1010 oh my god, this is sooo bad. Where can I opt-out of that AI slop please?

**Comment by adriankarlen** [comment]
Yes, this customizability seems to pop up more now that the theme has gotten more popular and I think this needs addressing. 

I am thinking that perhaps those icons should be somewhat grouped, based on the assumption that you either want to show them or hide them "all". Feels like an individual display variables simply becomes to cumbersome.
